Dockerfile

```diff
@@ -1,25 +1,16 @@
-FROM debian:buster-slim
+FROM alpine:latest 
```

Comment:
It's better to switch to the alpine based Node.js image itself.
Please update to `FROM node:lts-alpine` (and specify the version of the node package on top as a comment).